src/WattleScript.Interpreter/Tree/Statements/ForEachLoopStatement.cs

```diff
@@ -164,6 +172,66 @@ public override void Compile(FunctionBuilder bc)
 			bc.PopSourceRef();
 		}
 
+		private bool CompilePossibleLiteralRange(Loop l, FunctionBuilder bc)
```

Comment:
PossibleLiteral is not quite correct here, as the range itself is not literal expressions. Perhaps `CompileRangeStatement` ?

Reply:
Fixed in 8e870ee